.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  markdown-lint:
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-latest
```
We run other gh actions with `ubuntu-latest`, probably should do the same here?